core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -263,13 +263,18 @@ coreos:
       command: start
       content: |
         [Unit]
-        Requires={{.Etcd.SystemdUnitName}}
+        Wants={{.Etcd.SystemdUnitName}}
         After={{.Etcd.SystemdUnitName}}
 
         [Service]
-        Type=oneshot
+        Type=simple
+        Restart=on-failure
+        RestartSec=10
+
         EnvironmentFile={{.EtcdNodeEnvFileName}}
-        ExecStart=/opt/bin/cfn-signal
+        ExecStartPre=/usr/bin/systemctl is-active {{.Etcd.SystemdUnitName}}
+        ExecStartPre=/usr/bin/rkt fetch --pull-policy new {{.AWSCliImage.Options}}{{.AWSCliImage.RktRepo}}
+        ExecStart=-/opt/bin/cfn-signal
```

Comment:
If `/opt/bin/cfn-signal` failed after all images fetched, then CF don't expect this signal, it happens when node is rebooted lets say.  if CF itself is down and not receiving even expected signals, then I am not sure retrying will do any help :)